Dockerfile

```diff
@@ -1,6 +1,6 @@
-FROM amazonlinux:2018.03.0.20180827
+FROM amazonlinux:2
```

Comment:
Do we want to pin a specific version of AL2?

Reply:
I can't find specifically which version of AL2 is used by lambda, have you found anything?

Reply:
This just says "Custom"
https://docs.aws.amazon.com/lambda/latest/dg/lambda-runtimes.html

Reply:
I was thinking we could arbitrarily pick `2.0.20200722.0`, which is the latest tag on Docker Hub right now.